src/main/java/application/controllers/TreeController.java

```diff
@@ -131,14 +130,17 @@ public void colorSelectedStrains() {
      */
     public void applyCellHighlight(Cell cell) {
         if (cell instanceof LeafCell) {
-            List<Cell> parentList = new ArrayList<>();
-            parentList.add(cell);
+            String temp = ((LeafCell) cell).getName();
             collectedStrains.clear();
-            collectedStrains.add(cell);
 
-            applyColorUpwards(parentList,
-                    determineLinColor(metaData.get(((LeafCell) cell).getName())),
-                    4.0);
+            if (temp.contains("TKK")) {
+                List<Cell> parentList = new ArrayList<>();
+                parentList.add(cell);
+                collectedStrains.add(cell);
+                applyColorUpwards(parentList,
+                        determineLinColor(metaData.get(temp)),
```

Comment:
There is no need for this statement to be placed on three lines. Its length does not exceed the 100 chars.